liqd_lunch/frontend/src/ResistanceList.js

```diff
@@ -15,18 +15,21 @@ class  ResistanceList  extends  Component {
             isLoading: false
         };
         this.nextPage  =  this.nextPage.bind(this);
+        this.getItems = this.getItems.bind(this);
         this.setShowResult = this.setShowResult.bind(this);
-        this.timer = null;
+        this.itemTimer = null;
+        this.resultTimer = null;
     }
 
     componentDidMount() {
         this.getItems()
-        this.timer = setInterval(() => this.getItems(), 5000)
-        this.timer = setInterval(this.setShowResult, 1000)
```

Comment:
Haha, me and Philli were wondering yesterday why this even worked :) So it did not work and the getItems timer was never destroyed? If yes, where can you see that?